api/src/main/java/org/pmiops/workbench/cdr/model/Criteria.java

```diff
@@ -0,0 +1,235 @@
+        this.selectable = selectable;
+        return this;
+    }
+
+    @Column(name = "est_count")
+    public String getCount() {
+        return count;
+    }
+
+    public void setCount(String count) {
+        this.count = count;
+    }
+
+    public Criteria count(String count) {
+        this.count = count;
+        return this;
+    }
+
+    @Column(name = "concept_id")
+    public String getConceptId() {
+        return conceptId;
+    }
+
+    public void setConceptId(String conceptId) {
+        this.conceptId = conceptId;
+    }
+
+    public Criteria conceptId(String conceptId) {
+        this.conceptId = conceptId;
+        return this;
+    }
+
+    @Column(name = "domain_id")
+    public String getDomainId() {
+        return domainId;
+    }
+
+    public void setDomainId(String domainId) {
+        this.domainId = domainId;
+    }
+
+    public Criteria domainId(String domainId) {
+        this.domainId = domainId;
+        return this;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Criteria criteria = (Criteria) o;
+
+        if (id != criteria.id) return false;
+        if (sortOrder != criteria.sortOrder) return false;
+        if (parentId != criteria.parentId) return false;
+        if (group != criteria.group) return false;
+        if (selectable != criteria.selectable) return false;
+        if (type != null ? !type.equals(criteria.type) : criteria.type != null) return false;
+        if (code != null ? !code.equals(criteria.code) : criteria.code != null) return false;
+        if (name != null ? !name.equals(criteria.name) : criteria.name != null) return false;
+        if (count != null ? !count.equals(criteria.count) : criteria.count != null) return false;
+        if (conceptId != null ? !conceptId.equals(criteria.conceptId) : criteria.conceptId != null) return false;
+        return domainId != null ? domainId.equals(criteria.domainId) : criteria.domainId == null;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = (int) (id ^ (id >>> 32));
+        result = 31 * result + (int) (sortOrder ^ (sortOrder >>> 32));
+        result = 31 * result + (int) (parentId ^ (parentId >>> 32));
+        result = 31 * result + (type != null ? type.hashCode() : 0);
+        result = 31 * result + (code != null ? code.hashCode() : 0);
+        result = 31 * result + (name != null ? name.hashCode() : 0);
+        result = 31 * result + (group ? 1 : 0);
+        result = 31 * result + (selectable ? 1 : 0);
+        result = 31 * result + (count != null ? count.hashCode() : 0);
+        result = 31 * result + (conceptId != null ? conceptId.hashCode() : 0);
+        result = 31 * result + (domainId != null ? domainId.hashCode() : 0);
+        return result;
+    }
+
+    @Override
+    public String toString() {
+        final StringBuffer sb = new StringBuffer("Criteria{");
```

Comment:
Similarly, you could use ToStringBuilder here:

https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html

Reply:
sure